locker/AzureBackup/BackupServices.Tests/ScenarioTests/ContainerTests.cs

```diff
@@ -0,0 +1,80 @@
+﻿using Microsoft.Azure.Management.BackupServices;
```

Comment:
The locker folder got added by mistake by one of my teammate. I have removed it.